Signal/src/ViewControllers/HomeView/HomeViewCell.h

```diff
@@ -0,0 +1,23 @@
+//
+//  Copyright (c) 2018 Open Whisper Systems. All rights reserved.
+//
+
+NS_ASSUME_NONNULL_BEGIN
```

Comment:
I renamed & moved the home view classes into a new directory/project folder.  About half of the logic didn't change - you might get a better diff outside GH's web UI.